perma_web/api2/urls.py

```diff
@@ -0,0 +1,67 @@
+from django.conf.urls import url
+from rest_framework.routers import APIRootView
+
+from perma.urls import guid_pattern
+
+import views
```

Comment:
Curious about a triviality: in the perma urls.py, signals.py, models.py, and possibly elsewhere, when importing things from the current directory, it's done with a ./ (e.g. ./views or from ./views import blah blah blah). Is either one better style, so far as you know?